packages/x-components/src/plugins/devtools/utils.ts

```diff
@@ -0,0 +1,39 @@
+import { InspectorNodeTag } from '@vue/devtools-api';
+import { RootXStoreModule } from '../../store/x.module';
+import { NiladicFunction } from '../../utils/types';
+import { XModuleName } from '../../x-modules/x-modules.types';
+
+/** Unique text and background colors for each module. */
+type ModuleColors = Record<XModuleName, Pick<InspectorNodeTag, 'textColor' | 'backgroundColor'>>;
+/** Unique color values for the text and background of each module. */
+export const moduleColors: ModuleColors = Object.keys(
+  (RootXStoreModule.state as NiladicFunction)!()
+).reduce((colors, moduleName, index, keys) => {
+  const hue = Math.trunc((index * 360) / keys.length);
+  colors[moduleName as XModuleName] = {
+    textColor: hslToHex(hue, 50, 90),
```

Comment:
`50, 90, 85, 30` should you give some context about the why of this numbers? This king of magic numbers are dangerous.